src/Core/Algorithm/Subdivision/LoopSubdivider.hpp

```diff
@@ -0,0 +1,107 @@
+#ifndef RADIUMENGINE_LOOPSUBDIVIDER_H
+#define RADIUMENGINE_LOOPSUBDIVIDER_H
+
+#include <Core/Mesh/TopologicalTriMesh/TopologicalMesh.hpp>
+
+#include <OpenMesh/Tools/Subdivider/Uniform/SubdividerT.hh>
+
+namespace Ra {
+namespace Core {
+
+/// This class implements the Loop subdivision algorithm
+///
+/// This class extends OpenMesh's LoopT subdivider to handle attributes.
+/// \note We here consider that boundary halfedges do not store attributes.
+class RA_CORE_API LoopSubdivider
+    : public OpenMesh::Subdivider::Uniform::SubdividerT<TopologicalMesh, Scalar> {
+  public:
+    using base = OpenMesh::Subdivider::Uniform::SubdividerT<TopologicalMesh, Scalar>;
+    using Weight = std::pair<Scalar, Scalar>;
+    using Weights = std::vector<Weight>;
+
+  public:
+    LoopSubdivider() : base() { init_weights(); }
+
+    LoopSubdivider( TopologicalMesh& mesh ) : base( mesh ) { init_weights(); }
+
+    ~LoopSubdivider() {}
+
+  public:
+    /// must implement
```

Comment:
same as in CatmullClarkSubdivider.hpp. What is the meaning of this documentation?